rasa/core/evaluation/marker_base.py

```diff
@@ -347,7 +348,8 @@ def _split_sessions(events: List[Event]) -> List[Tuple[List[Event], int]]:
             )
             end_idx = session_start_indices[session_idx]
             session = [events[idx] for idx in range(start_idx, end_idx)]
-            sessions_and_start_indices.append((session, start_idx))
+            if len(session) > 0:
+                sessions_and_start_indices.append((session, start_idx))
```

Comment:
@ka-bu Are you happy with this fix or do you think there's a better way to do it?

Reply:
It would be better to change the parameter of the range and refactor variable names a bit to reflect - and prepend session start events to the tests. The current implementation assumes that a session start is issued when a new conversation starts but doesn't take into account the first one starts with that as well. I can just add that to this branch if that's ok?

Reply:
Yes no problem, if you want to fix it differently then go for it.